InvenTree/part/test_part.py

```diff
@@ -87,7 +87,7 @@ def test_mail_configured(self):
 
     def test_user_settings(self):
         result = inventree_extras.user_settings(self.user)
-        self.assertEqual(len(result), 36)
```

Comment:
Instead of hard-coding this number, latest master checks the length of the UserSetting.SETTINGS dict